src/options.c

```diff
@@ -260,120 +260,120 @@ static struct opt_entry opt_list[] = {
-"--users=[-]LOGIN|UID[,..]  [do not] load this (these) user(s) only\n" \
-"--groups=[-]GID[,..]       load users [not] of this (these) group(s) only\n" \
-"--shells=[-]SHELL[,..]     load users with[out] this (these) shell(s) only\n" \
-"--salts=[-]COUNT[:MAX]     load salts with[out] COUNT [to MAX] hashes, or\n" \
-"--salts=#M[-N]             load M [to N] most populated salts\n" \
-"--costs=[-]C[:M][,...]     load salts with[out] cost value Cn [to Mn]. For\n" \
+"--stress-test[=TIME]       Loop self tests forever\n" \
+"--test-full=LEVEL          Run more thorough self-tests\n" \
+"--no-mask                  Used with --test for alternate benchmark w/o mask\n" \
+"--skip-self-tests          Skip self tests\n" \
+"--users=[-]LOGIN|UID[,..]  [Do not] load this (these) user(s) only\n" \
+"--groups=[-]GID[,..]       Load users [not] of this (these) group(s) only\n" \
+"--shells=[-]SHELL[,..]     Load users with[out] this (these) shell(s) only\n" \
+"--salts=[-]COUNT[:MAX]     Load salts with[out] COUNT [to MAX] hashes, or\n" \
+"--salts=#M[-N]             Load M [to N] most populated salts\n" \
+"--costs=[-]C[:M][,...]     Load salts with[out] cost value Cn [to Mn]. For\n" \
 "                           tunable cost parameters, see doc/OPTIONS\n" \
 JOHN_USAGE_FORK \
-"--node=MIN[-MAX]/TOTAL     this node's number range out of TOTAL count\n" \
-"--save-memory=LEVEL        enable memory saving, at LEVEL 1..3\n" \
-"--log-stderr               log to screen instead of file\n"             \
-"--verbosity=N              change verbosity (1-%u or %u for debug, default %u)\n" \
-"--no-log                   disables creation and writing to john.log file\n"  \
-"--bare-always-valid=Y      treat bare hashes as valid (Y/N)\n" \
-"--config=FILE              use FILE instead of john.conf or john.ini\n" \
-"--encoding=NAME            input encoding (eg. UTF-8, ISO-8859-1). See also\n" \
+"--node=MIN[-MAX]/TOTAL     This node's number range out of TOTAL count\n" \
+"--save-memory=LEVEL        Enable memory saving, at LEVEL 1..3\n" \
+"--log-stderr               Log to screen instead of file\n"             \
+"--verbosity=N              Change verbosity (1-%u or %u for debug, default %u)\n" \
+"--no-log                   Disables creation and writing to john.log file\n"  \
+"--bare-always-valid=Y      Treat bare hashes as valid (Y/N)\n" \
+"--config=FILE              Use FILE instead of john.conf or john.ini\n" \
+"--encoding=NAME            Input encoding (eg. UTF-8, ISO-8859-1). See also\n" \
 "                           doc/ENCODINGS.\n" \
-"--input-encoding=NAME      input encoding (alias for --encoding)\n" \
-"--target-encoding=NAME     output encoding (used by format, see doc/ENCODINGS)\n" \
-"--force-tty                set up terminal for reading keystrokes even if we're\n" \
+"--input-encoding=NAME      Input encoding (alias for --encoding)\n" \
+"--target-encoding=NAME     Output encoding (used by format, see doc/ENCODINGS)\n" \
```

Comment:
I only now noticed we should move `--internal-codepage` up to between these two. Then the repetitions of "see doc/ENCODINGS" can be dropped.